examples/logger.py

```diff
@@ -8,6 +8,7 @@
 
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
+from PyQt5.QtWidgets import QComboBox
```

Comment:
Please merge this line into the above one!

If the line is too long, write it down as follows:
```python
from PyQt5.QtWidgets import (
    QWidget, QVBoxLayout, ...
)
```